main.tf

```diff
@@ -407,6 +408,19 @@ resource "kubernetes_deployment" "this" {
             "--aws-max-retries=10",
           ]
 
+          env {
```

Comment:
A dynamic block with optional environment variables would be a cleaner solution, I think.
Any chance that this is possible?